libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -25,10 +25,24 @@ void VotePacketHandler::process(const PacketData &packet_data, const std::shared
   LOG(log_dg_) << "Received PBFT vote " << vote_hash;
 
   const auto vote_round = vote->getRound();
+  const auto current_pbft_round = pbft_mgr_->getPbftRound();
+  // Check if reward vote
```

Comment:
what does it mean "check if reward vote" ? You should probably to start using grammar checker for all comments.

Reply:
fixed